packages/gatsby-source-graphql/README.md

```diff
@@ -47,21 +47,35 @@ module.exports = {
       },
     },
 
+    // Advanced config, using a custom fetch function
+    {
+      resolve: "gatsby-source-graphql",
+      options: {
+        typeName: "GitHub",
+        fieldName: "github",
+        url: "https://api.github.com/graphql",
+        // A `fetch`-compatible API to use when making requests.
+        fetch: (uri, options = {}) =>
+          fetch(uri, { ...options, headers: sign(options.headers) }),
+      },
+    },
+
     // Complex situations: creating arbitrary Apollo Link
     {
       resolve: "gatsby-source-graphql",
       options: {
         typeName: "GitHub",
         fieldName: "github",
         // Create Apollo Link manually. Can return a Promise.
-        createLink: (pluginOptions) => {
+        createLink: pluginOptions => {
           return createHttpLink({
-            uri: 'https://api.github.com/graphql',
+            uri: "https://api.github.com/graphql",
             headers: {
-              'Authorization': `Bearer ${process.env.GITHUB_TOKEN}`,
+              Authorization: `Bearer ${process.env.GITHUB_TOKEN}`,
```

Comment:
Can you please drop these changes? They seem unrelated.

I'll leave it to the @gatsbyjs/learning team to comment if necessary (but lgtm, thanks!).

Reply:
This doesn't seem to imply a meaningful change, just a formatting one. I'd drop these changes and focus on the addition above.

Reply:
This is prettier’s change. It fails lint otherwise.

Reply:
Looks like it wasn't previously run due to the missing bracket. Thanks for the catch.